package.json

```diff
@@ -5,11 +5,14 @@
   "dependencies": {
     "@material-ui/core": "^4.11.4",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.60",
     "@material-ui/styles": "^4.11.4",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "immutability-helper": "^3.1.1",
+    "lodash": "^4.17.21",
+    "postgresql": "^0.0.1",
     "react": "^17.0.2",
     "react-dnd": "^14.0.2",
     "react-dnd-html5-backend": "^14.0.0",
```

Comment:
Ultimately this is the move - since the package.jsons are managing separate React builds (and since I want to reference V1 for the near future), I'll hold off on merging them an we can just remove the entire V1 build later on (along with its package.json). And as you mentioned, maybe have a Client folder next to an API folder. 

Another complicating factor is that I'm using yarn rather than npm for V2. I think yarn used to be faster/more stable but npm may have caught up lately. Going forward we should stick to one or the other because apparently you can get your wires crossed by mixing the two (another reason I'd like to avoid merging these quite yet).